Makefile

```diff
@@ -33,7 +33,7 @@ starttestcontainer:
 		-p 9999:9999 \
 		--env-file "${HOME}/.pingidentity/config" \
 		-v $$(pwd)/server-profiles/shared-profile:/opt/in \
-		-v $$(pwd)/server-profiles/12.1/data.json:/opt/in/instance/bulk-config/data.json \
+		-v $$(pwd)/server-profiles/12.2/data.json:/opt/in/instance/bulk-config/data.json \
 		pingidentity/pingfederate:latest
```

Comment:
Might be best to make this tied to either a docker-builds release or a PF version, to avoid confusion if a cached `latest` is present in docker

Reply:
PF version pinned to 12.2.0-latest